include/swift/Basic/LLVM.h

```diff
@@ -64,6 +65,7 @@ namespace swift {
   using llvm::SmallPtrSet;
   using llvm::SmallString;
   using llvm::StringRef;
+  using llvm::StringLiteral;
```

Comment:
Nitpick: alphabetize!

Reply:
I see that, but given that you chose to put this next to StringRef instead of at the end it's weird that you chose to put it *after* StringRef. (I'm okay with you not realphabetizing the whole thing.)

Reply:
Ok. So you are arguing for not making the problem worse... ok.